### ISSUE TITLE
Should the global action buttons insist upon aria-controls?

### ISSUE THREAD

**Issue by lee-chase** [carbon-design-system/carbon-components-vue#476]
It makes sense for the global action buttons to have an aria-controls if they only ever control a panel. If they do not have to have a panel then it is only needed for those that do control a panel.

**Comment by lee-chase** [comment]
Aria-controls should be optional as the global action can be search or non-panel controlling button.
